lib/auth-client.js

```diff
@@ -73,7 +77,16 @@ function AuthClient(options) {
   this.clientSecret = options.clientSecret;
   this.redirectUri = options.redirectUri;
   this.scope = options.scope;
-  this.development = options.development || false;
+
+  if (options.development !== undefined) {
+    /* eslint-disable-next-line no-console */
+    console.warn(`DeprecationWarning: development flag is deprecated.
```

Comment:
nit: for future warnings we should make use of the builtin [`process.emitWarning`](https://nodejs.org/api/process.html#process_process_emitwarning_warning_options)